README.md

```diff
@@ -0,0 +1,107 @@
+
+# 📚📙📖 Online Bookstore API 📖📙📚
+This is an application for online book store . 
+An Online Bookstore API is a back-end part of virtual store  where customers can browse the catalog and select books of interest. User can select many books and those books stored in cart. At checkout time, the items in the shopping cart will be presented as an order.
+Admins can manage this online bookstore.  This application serves as the core infrastructure , delivering crucial functionalities for book management, user authentication, and order processing.
+
+ ### 🛠️ Technologies and Tools Used
+- Spring Boot: For building the application framework.
+- Spring Security: To handle authentication and authorization.
+- Spring Data JPA: For database interactions using JPA.
+- Spring Web: For building web-based applications with Spring MVC.
+- Swagger: For API documentation and testing.
+- Hibernate: As the ORM tool.
+- MySQL: For the relational database.
+- Liquibase: For database schema versioning and management.
+- Docker: For containerizing the application.
+- Maven: For project management and dependency management.
+### 🚀 Current Functionalities
+- ##### Authentication 
+  - Register a new user: POST /api/auth/register
+  - Authenticate a user: POST /api/auth/login
```

Comment:
let's provide more information on the auth topic, give examples of requests & responses of those endpoints and explain what to do with the token that we received. Also provide user and admin credentials here as well